docs/installation.rst

```diff
@@ -30,22 +19,30 @@ Prerequisites
 
 1. Linux x64.
 2. |driver link|_ supporting `CUDA 10.0 <https://developer.nvidia.com/cuda-downloads>`__ or later (i.e., 410.48 or later driver releases).
-3. One or more of the following deep learning frameworks:
+3. (Optional) One or more of the following deep learning frameworks:
 
   - |mxnet link|_ ``mxnet-cu100`` or later.
   - |pytorch link|_ or later.
   - |tf link|_ or later.
 
 
-Installation
-^^^^^^^^^^^^
+NGC Containers
+--------------
 
-Execute the following command to install the latest DALI for specified CUDA version (please check
-:doc:`support matrix <support_matrix>` to see if your platform is supported):
+DALI is preinstalled in the TensorFlow, PyTorch, and MXNet containers in versions 18.07 and
+later on `NVIDIA GPU Cloud <https://ngc.nvidia.com>`_.
 
-.. note::
+----
+
+pip
```

Comment:
I'm missing something like: `install DALI releases` or `official DALI wheels`.
`pip` lacks the meaning in the navigation on the right.